packages/cli/tests/e2e/apim/TestUseExistingApim.tests.ts

```diff
@@ -26,8 +26,13 @@ describe("Use an existing API Management Service", function () {
   const projectPath = path.resolve(testFolder, appName);
   const existingRGName = `${appName}existing`;
   const existingRGNameExtend = `${existingRGName}-rg`;
+  process.env.SIMPLE_AUTH_SKU_NAME = "B1";
```

Comment:
Please deep copy process.env and use the deep copied one. This will influence other cases and may result in unknown issues.

Reply:
Got it.